### PR TITLE
Remove incomplete example used here

### PR DESCRIPTION
**Pull request by MovieStoreGuy**
Had added an incomplete example to this and removing it from the contribution guide.

### CODE REVIEW

**Review by mx-psi — COMMENTED** [review]
Maybe a good PR to address https://github.com/open-telemetry/opentelemetry-collector/pull/4702#pullrequestreview-857897827

**Comment by bogdandrutu** [comment]
@MovieStoreGuy please address all the comments from https://github.com/open-telemetry/opentelemetry-collector/pull/4702

**Comment by MovieStoreGuy** [comment]
No worries, sorry I had gotten sidetrakced trying to fix an issue within my team.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -196,10 +195,10 @@ To keep naming patterns consistent across the project, naming patterns are enfor
 In order to simplify developing within the project, library recommendations have been set
 and should be followed.
 
-| Scenario | Recommended                                   | Rationale        |
-| ----------------------------------------------------------------------------|
-| Hashing  | ["hashing/fnv"](https://pkg.go.dev/hash/fnv)  | The project adopted this as the default hashing method due to the efficiency and is reasonable for non cryptographic use |
-| Testing  | Use `t.Parallel()` where possible             | Enabling more test to be run in parallel will speed up the feedback process when working on the project.                 |
+| Scenario 	| Recommended                                  	| Recommended                                                                                                              	|
```
Comment (codeboten) [inline CONTRIBUTING.md:198]: This header now contains Recommended twice, is this expected?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -177,16 +177,15 @@ To keep naming patterns consistent across the project, naming patterns are enfor
 - Methods that return a variable that uses the zero value or values provided via the method MUST have the prefix `New`. For example:
   - `func NewKinesisExporter(kpl aws.KinesisProducerLibrary)` allocates a variable that uses
     the variables passed on creation.
-  - `func NewKeyValueBuilder()` may allocate internal varialbes to a safe zero value.
-- Methods that return a variable that uses non zero value(s) that impacts business logic must use `NewDefault`. For example:
+  - `func NewKeyValueBuilder()` shall allocate internal variables to a safe zero value.
```
Comment (codeboten) [inline CONTRIBUTING.md:180]: Should `shall` be capitalized as well?

Comment (codeboten) [reply CONTRIBUTING.md:180]: >MUST   This word, or the terms "REQUIRED" or "SHALL", mean that the
   definition is an absolute requirement of the specification.

Based on ^^^ from the RFC, I would expect shall to be interchangeable with must, but i dont have a strong preference for one or the other.

Comment (bogdandrutu) [reply CONTRIBUTING.md:180]: long debate between must vs should. If there is no exception then we should use MUST, if there may be an edge case then we should use SHOULD

File: CONTRIBUTING.md (hunk 1 above)
Comment (MovieStoreGuy) [reply CONTRIBUTING.md:198]: No... That was my oversight of trying to use an online formatter because I never remember what markdown engine wants.

**Review by bogdandrutu — APPROVED** [review]

**Review by codeboten — APPROVED** [review]

**Pull request merged**